stable/rocketchat/values.yaml

```diff
@@ -2,8 +2,8 @@
 ## ref: https://hub.docker.com/r/rocketchat/rocket.chat/tags
 ##
 image:
-  repository: docker.io/rocketchat/rocket.chat
-  tag: 2.4.9
+  repository: rocketchat/rocket.chat
+  tag: 3.2.1
```

Comment:
I believe 3.2.2 will be out today or tomorrow.  So might delay so latest includes a few fixes that you’d want :)

Reply:
Oh yes, thanks!

Reply:
perfect thanks! :)